src/main.cpp

```diff
@@ -33,7 +33,7 @@ void startingSensors() {
     sensors.setOnErrorCallBack(&onSensorDataError);
     sensors.setSampleTime(cfg.stime);             // config sensors sample time
     sensors.setDebugMode(false);                  // [optional] debug mode
-    sensors.init(cfg.getSensorType());            // start all sensors and
+    sensors.init(sensors.Mhz19);            // start all sensors and
```

Comment:
please for the PR restore the default parameter.

Reply:
done